Detectors/TRD/base/include/TRDBase/PadPlane.h

```diff
@@ -71,9 +74,25 @@ class PadPlane
 
   GPUd() int getPadRowNumber(double z) const;
   GPUd() int getPadRowNumberROC(double z) const;
+  GPUd() double getPadRow(double z) const;
   GPUd() int getPadColNumber(double rphi) const;
+  GPUd() double getPad(double y, double z) const;
 
-  GPUd() double getTiltOffset(double rowOffset) const { return mTiltingTan * (rowOffset - 0.5 * mLengthIPad); };
+  GPUd() double getTiltOffset(double rowOffset) const
+  // This function assumes that the outer pads are not tilted about their center
```

Comment:
I don't think this is correct. For the outer pads also an offset is returned, it is just not fully correct. Why don't you also uncomment the function below which takes care also of the outer pads?
Btw I cannot find the diagram you are referring to in the TDR on page 28.

Reply:
Yes, I think it can only be correct if the outer pads are not tilted about their centers. I don't know if this is the case or not. Perhaps the person who wrote this function was aware of something we are not and I'm not sure where to check this.

The commented function below it is what I wrote to replace it. Need to determine how the outer pads are titled before we know which calculation is correct.

I was referring to figure 4.12 in the TDR